quarkchain/cluster/shard_state.py

```diff
@@ -1995,4 +2015,23 @@ def get_total_balance(
         limit: int,
         starter: Optional[bytes] = None,
     ) -> Tuple[int, bytes]:
-        return 42, b""
+        """
+        starter should be exclusive
+        """
+        evm_state = self._get_evm_state_from_hash(block_hash)
+
+        trie = evm_state.trie.trie
+        key = trie.next(bytes(32) if not starter else starter)
+        total = 0
+        ret = None
+        while limit > 0 and key is not None:
+            addr = evm_state.trie.db.get(key)
```

Comment:
add a comment this is how to convert trie hash back to actual key from `SecureTrie`